File: Sakila_Net/Sakila_Net/CS_SakilaLib/DAO/DAOFilmCategory.cs
//using System;
using System.Data;
//using System.Collections.Generic;
//using System.Configuration;
//using System.Collections;
using Cl.Harrys.Sakila.Model;
namespace  Cl.Harrys.Sakila.DAO
{   
  public class DAOFilmCategory 
  {
       public static DataTable sqlSelectAll(int nPagina,int cantidad)
	   {
           return DAOBD.DAOFilmCategory.selectAll(nPagina,cantidad);
	   }
       public static bool sqlSelect(ClFilmCategory filmCategory)
	   {
            return DAOBD.DAOFilmCategory.SelectOne(filmCategory); 
	   }
	   public static int sqlInsert(ClFilmCategory filmCategory)
	   {
            return DAOBD.DAOFilmCategory.sqlInsert(filmCategory);
       }
	   public static int sqlDelete(ClFilmCategory filmCategory)
	   {
            return DAOBD.DAOFilmCategory.sqlDelete(filmCategory);
       }
	   public static int sqlUpdate(ClFilmCategory filmCategory)
	   {
            return DAOBD.DAOFilmCategory.sqlUpdate(filmCategory);
       }
   }
}

File: Sakila_Net/VB_SakilaLib/DAO/DAOFilmCategory.vb
'using System;
Imports  System.Data
'using System.Collections.Generic;
'using System.Configuration;
'using System.Collections;
Imports  SakilaLib.Cl.Harrys.Sakila.Model
'using Cl.Harrys.BD;

Namespace Cl
    Namespace Harrys
        Namespace Sakila
            Namespace DAO

                Public Class DAOFilmCategory
                    'Inherits SakilaLib.Cl.Harrys.Sakila.DAOBD.DAOFilmCategory
                    Public Shared  Function SelectAll(nPagina as Integer ,cantidad as Integer) as DataTable
                        Return DAOBD.DAOFilmCategory.SelectAll(nPagina, cantidad)	   
                    End Function
                    Public Shared  Function SelectOne( filmCategory as ClFilmCategory) as Boolean
                        Return DAOBD.DAOFilmCategory.SelectOne(filmCategory)     
                    End Function
                    Public Shared  Function SqlInsert( filmCategory as ClFilmCategory) as Integer
                        Return DAOBD.DAOFilmCategory.SqlInsert(filmCategory)     
                    End Function
                    Public Shared  Function SqlDelete( filmCategory as ClFilmCategory) as Integer
                        Return DAOBD.DAOFilmCategory.SqlDelete(filmCategory)     
                    End Function
                    Public Shared  Function SqlUpdate( filmCategory as ClFilmCategory) as Integer
                        Return DAOBD.DAOFilmCategory.SqlUpdate(filmCategory)     
                    End Function
                End Class

            End Namespace  
        End Namespace  
    End Namespace  
End Namespace  
